lib/commands/deploy.ts

```diff
@@ -81,10 +81,12 @@ the image in the balenaCloud build servers.)
 Unless an image is specified, this command will look into the current directory
 (or the one specified by --source) for a docker-compose.yml file.  If one is
 found, this command will deploy each service defined in the compose file,
-building it first if an image for it doesn't exist. If a compose file isn't
-found, the command will look for a Dockerfile[.template] file (or alternative
-Dockerfile specified with the \`-f\` option), and if yet that isn't found, it
-will try to generate one.
+building it first if an image for it doesn't exist. Image names will be looked
+up according to the scheme: <projectName>_<serviceName>.
```

Comment:
The CI tests are failing because this line is not matching between `deploy.ts` and `cli.markdown` -

```
diff --git a/doc/cli.markdown b/doc/cli.markdown
index 4f87ee7a..4ee75d51 100644
--- a/doc/cli.markdown
+++ b/doc/cli.markdown
-up according to the scheme: `<projectName>_<serviceName>`.
+up according to the scheme: <projectName>_<serviceName>.

** Uncommitted changes found (^^^ diff above ^^^) - FAIL **
Error: "catch-uncommitted": Error: npx failed with exit code 1:
```
https://ci.balena-dev.com/builds/1082955

Reply:
fixed o/